frontend/src/service/Change.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Err, ErrorService } from './Error.service';
+import { ChangeGraphQLApi } from './shortGraphQL/ChangeGraphQL.api';
+import { Change } from '../entity/Change';
+
+export class ChangeService {
+  constructor(
+    private changeGraphQLApi: ChangeGraphQLApi,
+    private errorService: ErrorService
+  ) {}
+
+  createChange(title: string, summaryMarkdown: string): Promise<Change> {
+    return new Promise<Change>((resolve, reject) => {
+      this.changeGraphQLApi
+        .createChange(title, summaryMarkdown)
+        .then(resolve)
+        .catch(errCode => {
+          if (errCode === Err.Unauthenticated) {
+            reject({ authenticationErr: 'User is not authenticated' });
```

Comment:
I wonder whether we should hard code the string here. What are their purpose?

Reply:
@byliuyang This was the format being used in the [ShortLinks service](https://github.com/short-d/short/blob/master/frontend/src/service/Url.service.ts#L73). Should be try to refactor to a generic one everywhere?

Reply:
@rohithbalaji123 Let’s refactor & clean up everything!

Reply:
@byliuyang , we should come up with a definite error handling mechanism. The lines [here](https://github.com/short-d/short/blob/master/frontend/src/service/Url.service.ts#L71) and [here](https://github.com/short-d/short/blob/master/frontend/src/service/ShortLink.service.ts#L30) handles error in a way, whereas [this](https://github.com/short-d/short/blob/master/frontend/src/service/ShortLink.service.ts#L42) in a different way. Which are we finalizing on?

Reply:
@rohithbalaji123 Can we add a TODO here so that we look into it later without blocking this PR?